cli/defenseclaw/file_permissions.py

```
@@ -942,33 +954,251 @@ def _verify_or_repair_windows_private_target(path: str) -> None:
+                    cleanup_problem = _safe_os_error_label(exc)
+                else:
+                    delete_requested = True
+    finally:
+        try:
+            os.close(descriptor)
+        except OSError as exc:
+            close_problem = _safe_os_error_label(exc)
+
+    if problem is None or repair_problem is None:
+        if close_problem is not None:
+            raise OSError(f"managed Windows publication handle close failed: {close_problem}") from None
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)